### PR TITLE
Closes #3: Add RDS footer logos

### LINKED ISSUE #3: Add RDS footer logos

### PR DESCRIPTION
**Pull request by sudmar**
Closes #3

### CODE REVIEW

File: assets/logos/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ # Installing logos
+
+ - Copy all images to the specified location:
+
+        [glassfish_directory]/domains/domain1/docroot/logos/
+
+
+- Configure your Dataverse installation to make use of the supplied `footer_logos_rds.html` file:
+
+        curl -X PUT -d 'footer_logos_rds.html' [dataverse_web_address]/api/admin/settings/:FooterCustomizationFile
```
Comment (MrK191) [inline assets/logos/README.md]: right now we are using file based settings so this shouldn't be done via API. I think I overlooked it on other pr.

**Review by MrK191 — CHANGES_REQUESTED** [review]

File: assets/logos/README.md (hunk 1 above)
Comment (sudmar) [reply assets/logos/README.md]: Okay, so how this should be done properly?
I took this method from the Dataverse docs, I don't know what exactly differs in our case.

Comment (MrK191) [reply assets/logos/README.md]: We are using file based settings that are in "dataverse.default.properties" normally you move those to ~/.dataverse and rename it to dataverse.properties then you edit the values. so in this case `FooterCustomizationFile=footer_logos_rds.html`.

Comment (sudmar) [reply assets/logos/README.md]: May I ask you for some further assistance?

I've copied the `dataverse.default.properties` to `~/.dataverse` and renamed it, and the changes don't seem to take effect. Do I need to do something else as well?

Comment (MrK191) [reply assets/logos/README.md]: Hmm, I'm looking at the code and this is the path `System.getProperty("user.home") + "/.dataverse/dataverse.properties"` so it extracts you'r home directory and concants the strings. It also overrides the default values.

Comment (MrK191) [reply assets/logos/README.md]: If this doesn't help we would need to debug together.

**Review by MrK191 — APPROVED** [review]

**Pull request merged**